daisy_workflows/image_import/windows/translate.ps1

```diff
@@ -265,25 +271,28 @@ function Install-32bitPackages {
 
 try {
   Write-Output 'Translate: Beginning translate PowerShell script.'
-  $script:outs_dir = Get-MetadataValue -key 'daisy-outs-path'
-  $script:install_packages = Get-MetadataValue -key 'install-gce-packages'
-  $script:sysprep = Get-MetadataValue -key 'sysprep'
-  $script:is_byol = Get-MetadataValue -key 'is_byol'
-  $script:is_x86 = Get-MetadataValue -key 'is_x86'
 
   Remove-VMWareTools
   Change-InstanceProperties
   Configure-Network
   Setup-NTP
   Configure-RDPSecurity
 
-  if ($script:is_x86.ToLower() -ne 'true') {
+  $script:install_packages = Get-MetadataValue -key 'install-gce-packages'
+  $script:sysprep = Get-MetadataValue -key 'sysprep'
+  $script:is_byol = Get-MetadataValue -key 'is_byol'
+  if ($script:install_packages -eq $null -or $script:sysprep -eq $null -or $script:is_byol -eq $null) {
```

Comment:
it would be better to know why reboot is the solution. It sounds more like a bug if reboot is occasionally required. Maybe you can dig into and found a big hole.
Previously we saw calling to metadata service failed, but I don't remember in any case it became null. Maybe the powershell lib doesn't work as expected (for example, handle metadata service error incorrectly?)

Reply:
Thanks for the feedback. I dug in a bit further and realized that specifically the vmware tools removal and reboot could have been the underlying issue. Which meant that the variables were read right at boot potentially before the network was fully ready which causes one of the variables to return a non-200 

The reboot approach was inspired by another customer issue where the network was unusable until the system was rebooted. It's meant to be a last ditch effort to help the import success by giving it one more fresh attempt instead of just aborting.